ios/Runner/Info.plist

```diff
@@ -5,8 +5,8 @@
 	<key>CFBundleLocalizations</key>
 	<array>
 		<string>en</string>
+		<string>es</string>
 		<string>pt</string>
-		<string>de</string>
```

Comment:
Forget to remove de translation from ios, related: https://github.com/breez/breezmobile/pull/614